src/controller/handleCountInput.js

```diff
@@ -0,0 +1,18 @@
+import { ERROR, LIMIT } from '../util/constants.js';
+import playRace from './playRace.js';
+
+function isValidCount(inputCount) {
+  if (+inputCount < LIMIT.MIN_LENGTH) {
+    alert(ERROR.MIN_COUNT);
+    return false;
+  }
+  return true;
+}
+
+export default function handleCountInput() {
+  const inputCount = document.querySelector('.count-input').value;
```

Comment:
```suggestion
  const { value } = document.querySelector('.count-input');
```
구조분해할당해서 사용해보는건 어떨까요?

Reply:
구조분해할당을 생각못했네요! 좋아요!!👍👍